subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -98,6 +98,24 @@ static int init_reset(void);
  * @return Pointer to lll_df_adv_cfg or NULL if there is no more free memory.
  */
 static struct lll_df_adv_cfg *df_adv_cfg_acquire(void);
+
+/*
+ * @brief Function updates content of cte_info field in periodic advertising chain.
```

Comment:
Description says `updates` but the name says `set`. I guess these should be aligned.

Reply:
Aligned to the name of the function. Also I've moved the header to function definition, to align it with other functions.